server/app.js

```diff
@@ -8,6 +8,8 @@ const routes = require("./routes/v1");
 
 const app = express()
 
+const SERVER_PORT = process.env.SERVER_PORT;
```

Comment:
Added separate variable because, eventually I think we need to validate these env vars based on environment that the application is going to run in. Ex: development, staging, production etc.

Reply:
I think we don't need this, if we're gonna merge this PR #6